server/common/utils/helpers.test.js

```diff
@@ -0,0 +1,391 @@
+    notExpected.map(property =>
+      result.map(cohort => expect(cohort).not.toHaveProperty(property))
+    );
+  });
+});
+
+describe('function responseWithCohort', () => {
+  const userId = ObjectId();
+  const cohort = cohortsMock[0];
+  const result = responseWithCohort(cohort, userId);
+
+  it('should return cohort data with desired properties', () => {
+    expectedCohortMetaDataProperties.map(property =>
+      expect(result).toHaveProperty(property)
+    );
+  });
+
+  const notExpected = ['favIds', 'ownerIds', 'memberIds'];
+
+  it('should return cohort data without favIds, memberIds, ownerIds', () => {
+    notExpected.map(property => expect(result).not.toHaveProperty(property));
+  });
+});
+
+describe('function checkIfArrayContainsUserId', () => {
+  const idsArray = ['123', '456', '789'];
+
+  it('checks if the passed array does contains the userID', () => {
+    const userId = '123';
+    const result = checkIfArrayContainsUserId(idsArray, userId);
+
+    expect(result).toBe(true);
+  });
+
+  it('checks if the passed array does not contain the user id', () => {
+    const userId = '999';
+    const result = checkIfArrayContainsUserId(idsArray, userId);
+
+    expect(result).toBe(false);
+  });
+
+  it('checks if the passed array does not contain the user id', () => {
+    const userId = ObjectId();
+    const idsArray = [ObjectId(), ObjectId(), ObjectId()];
+    const result = checkIfArrayContainsUserId(idsArray, userId);
+
+    expect(result).toBe(false);
+  });
+});
+
+describe('function checkIfGuest', () => {
+  const memberIds = ['123', '456', '789'];
+
+  it('checks if the passed array does not contain user id', () => {
+    const userId = ObjectId();
+    const result = checkIfGuest(memberIds, userId);
+
+    expect(result).toBe(true);
+  });
+
+  it('checks if the passed array does contain user id', () => {
+    const userId = '123';
+    const result = checkIfGuest(memberIds, userId);
+
+    expect(result).toBe(false);
+  });
+});
+
+describe('function responseWithCohortMembers', () => {
+  const ownerIds = ['123', '456', '789'];
+  const members = responseWithCohortMembers(usersMock, ownerIds);
+
+  it('should return objects with desired properties', () => {
+    expectedUsersProperties.map(property =>
+      members.map(member => expect(member).toHaveProperty(property))
+    );
+  });
+
+  it('should return user objects without sensitive data like email', () => {
+    const notExpectedUserProperties = ['email'];
+
+    notExpectedUserProperties.map(property =>
+      members.map(member => expect(member).not.toHaveProperty(property))
+    );
+  });
+});
+
+describe('function checkIfCohortMember', () => {
+  const cohort = cohortsMock[0];
+
+  it('should return true if passed user id is cohort member', () => {
+    const userId = ObjectId();
+    const result = checkIfCohortMember(cohort, userId);
+
+    expect(result).toBe(false);
```

Comment:
if method should return true when user is a cohort member why this test result should be false?